### PR TITLE
Series resample util

### PR DESCRIPTION
**Pull request by JensGM**

### CODE REVIEW

File: camille/util/resample.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+def resample(tseries, onto=None, interp='linear'):
+    try:
+        idx = onto.index
+    except:
+        idx = onto
+
+    ts = (
+        pd.concat([tseries, pd.Series(index=idx)])
+        .sort_index(kind='mergesort')
```
Comment (sveinung-r) [inline camille/util/resample.py]: Why not quicksort(default)?

Comment (JensGM) [reply camille/util/resample.py]: The sort must be stable. When we remove duplicate indices (before reindexing). We keep the first occurance of data, that should come from the function we are resampling, which is first in the series before the sorting

File: camille/util/resample.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        )
+
+    if interp in _pandas_supported_interps:
```
[the remark this answers is not in the document]
Comment (JensGM) [reply camille/util/resample.py]: Good idea

[the remark this answers is not in the document]
Comment (sveinung-r) [reply camille/util/resample.py]: Or pass interp with empty dataframes to test if valid

**Pull request merged**